### PR TITLE
Organizes WebSocket schema objects

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

In the previous PR (#850), the objects needed for WebSocket requests and responses started to pile up and get a bit messy. This PR is cleaning all that up. 

### Context of Change

Neater/better named objects

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After

Objects have better names and don't overlap so much.

## Test Plan

CI passes.

### CODE REVIEW

File: test/XRP/web-socket-network-client.test.ts [hunk 1]
```diff
@@ -110,7 +110,7 @@ describe('WebSocket Tests', function (): void {
         address,
         'subscribe_transaction_' + address,
         // eslint-disable-next-line @typescript-eslint/no-empty-function
-        (_data: WebSocketTransactionResponse) => {},
+        (_data: TransactionResponse) => {},
```
Comment (tedkalaw) [inline test/XRP/web-socket-network-client.test.ts:113]: will the compiler complain if you just remove the param? ie just change this to `() => {}`

File: src/XRP/shared/rippled-web-socket-schema.ts [hunk 2]
```diff
@@ -18,24 +18,29 @@ enum WebSocketReadyState {
...
+}
+
 /**
  * The standard format for a `subscribe` request to the Web Socket API.
  * @see https://xrpl.org/subscribe.html
  */
-interface SubscribeRequest {
-  id: number | string
-  command: RippledMethod
+interface SubscribeRequest extends BaseRequest {
```
Comment (tedkalaw) [inline src/XRP/shared/rippled-web-socket-schema.ts:43]: can you do

```
command: RippledMethod.subscribe
```
? 

if you do that and add `command: RippledMethod.accountLines` to AccountLinesRequest, ts will be able to tell the difference between `BaseRequest` objects if you switch on `BaseRequest.command`, which is pretty dope

**Review by tedkalaw — APPROVED** [review]
super dope. one thought inline about types

File: src/XRP/shared/rippled-web-socket-schema.ts (hunk 2 above)
Comment (mvadari) [reply src/XRP/shared/rippled-web-socket-schema.ts:43]: Typescript isn't happy when I try to do that and use the objects - `type RippledMethod is not assignable to type RippledMethod.subscribe`

**Review by amiecorso — APPROVED** [review]

**Pull request merged**